apps/web/src/components/nav/RootNavMenu.tsx

```diff
@@ -56,7 +56,7 @@ export const RootNavMenu: React.FC = () => {
         <NavMenuLinkButton
           label="Settings"
           icon={<IconSettings />}
-          link={ROUTES.SETTINGS}
+          link={ROUTES.PROFILE}
```

Comment:
redirect directly to the user profile page route, otherwise react router will do the navigation twice first to `/settings` and then to `/settings/profile`